SUPPORT.rst

```diff
@@ -4,14 +4,10 @@ Contact/Getting Help
 There are several ways to contact us or to get help:
 
 
-#. 
-   `\ **FAQ** <https://github.com/mozilla/DeepSpeech/wiki#frequently-asked-questions>`_ - We have a list of common questions, and their answers, in our `FAQ <https://github.com/mozilla/DeepSpeech/wiki#frequently-asked-questions>`_. When just getting started, it's best to first check the `FAQ <https://github.com/mozilla/DeepSpeech/wiki#frequently-asked-questions>`_ to see if your question is addressed.
+#. `FAQ <https://github.com/mozilla/DeepSpeech/wiki#frequently-asked-questions>`_ - We have a list of common questions, and their answers, in our `FAQ <https://github.com/mozilla/DeepSpeech/wiki#frequently-asked-questions>`_. When just getting started, it's best to first check the `FAQ <https://github.com/mozilla/DeepSpeech/wiki#frequently-asked-questions>`_ to see if your question is addressed.
```

Comment:
Do we really want to keep the wiki? I feel like nobody reads it.

Reply:
Yeah, I was thinking about that, we don't check or maintain the FAQ often enough. Should probably just tell people to search Discourse and the issue tracker.